src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/UserAcspMembership.java

```diff
@@ -211,11 +210,68 @@ public ResponseEntity<List<AcspMembership>> getAcspMembershipForUserId(
     return new ResponseEntity<>(memberships, HttpStatus.OK);
   }
 
-  @Override
-  public ResponseEntity<Void> updateAcspMembershipForId(
-      @NotNull String xRequestId,
-      @Pattern(regexp = "^[a-zA-Z0-9]*$") String id,
-      @Valid RequestBodyPatch requestBodyPatch) {
-    return null; // TODO(https://companieshouse.atlassian.net/browse/IDVA6-1147)
+  private void toBadRequestWhenActionIsNotPermitted( final String xRequestId, final AcspMembersDao requestingAcspMember, final AcspMembersDao targetAcspMembership ){
+    final String requestingUserId = requestingAcspMember.getUserId();
+    final UserRoleEnum requestingUserRole = requestingAcspMember.getUserRole();
+    final String targetUserId = targetAcspMembership.getUserId();
+    final UserRoleEnum targetUserRole = targetAcspMembership.getUserRole();
+
+    if ( requestingUserRole.equals( UserRoleEnum.STANDARD ) ){
```

Comment:
I've seen people commonly do:
```suggestion
    if ( UserRoleEnum.STANDARD.equals( requestingUserRole ) ){
```

As this would prevent a null pointer since the enum won't even be `null`, but I don't think any of your arguments are valid to be `null` - so probably not worth changing

Reply:
In the current context, it doesn't matter. But if I make the change you are suggesting, it could improve reusability/make the code more robust if there are changes in the future, so I think you make a good point. I've made the change.